plugin.video.infowars/default.py

```diff
@@ -311,35 +318,105 @@ def PlayURL(url):
+    #xbmcplugin.addDirectoryItem(int(sys.argv[1]), li_url, listitem,isFolder=True)
     #_addon.add_directory({'mode': 'NightlyNewsSubMenu','title':'Infowars Nightly News'},{'title':  cFL_('Infowars Nightly News','red')},is_folder=True,img=_artIcon,fanart=_artFanart)
-    _addon.add_directory({'mode': 'ClipsSubMenu','title':'Infowars Nightly News'},{'title':  cFL_('Infowars Clips','red')},is_folder=True,img=_artIcon,fanart=_artFanart)
+    _addon.add_directory({'mode': 'ClipsSubMenu','title':'Infowars Clips'},{'title':  cFL_('Infowars Clips','yellow')},is_folder=True,img=_artIcon,fanart=_artFanart)
     _addon.add_directory({'mode': 'DocSubMenu','title':'Acclaimed Documentaries'},{'title':  cFL_('Acclaimed Documentaries','blanchedalmond')},is_folder=True,img=_artIcon,fanart=_artFanart)
-    _addon.add_directory({'mode': 'HistoricShowsSubMenu','title':'Past Alex Jones Shows(video)'},{'title':  cFL_('Past Alex Jones Shows (Video)','yellow')},is_folder=True,img=_artIcon,fanart=_artFanart)
-    _addon.add_directory({'mode': 'HistoricShowsAudioSubMenu','title':'Past Alex Jones Shows(video)'},{'title':  cFL_('Past Alex Jones (Audio)','yellow')},is_folder=True,img=_artIcon,fanart=_artFanart)
-    _addon.add_directory({'mode': 'PaulJosephWatsonSubMenu','title':'Paul Joseph Watson (Video)'},{'title':  cFL_('Paul Joseph Watson (Video)','blue')},is_folder=True,img=_artIcon,fanart=_artFanart)
+    _addon.add_directory({'mode': 'HistoricShowsSubMenu','title':'Past Alex Jones Shows(video)'},{'title':  cFL_('Past Alex Jones Shows (Video)','yellow')},is_folder=True,img=_AJSIcon,fanart=_artFanart)
+    _addon.add_directory({'mode': 'HistoricShowsAudioSubMenu','title':'Past Alex Jones Shows(video)'},{'title':  cFL_('Past Alex Jones (Audio)','yellow')},is_folder=True,img=_AJSIcon,fanart=_artFanart)
+    
     eod()
 
+def Info_Wars_Live_Sub_Menu(title=''): #The Main Menu
+    WhereAmI('@ Info Wars Live')
+    
+    url = 'https://www.infowars.com/watch-alex-jones-show/'
+    response = urllib2.urlopen(url)
+    if response and response.getcode() == 200:
+        content = response.read()
+        videos= find_multiple_matches(content,"<iframe(.*?)</iframe>")
+        sources = []
+        temp = 0
+        for entry in videos: 
+            video_id = find_single_match(entry,"src=\"https://www.youtube.com/embed/(.*?)\"")
+            url = "plugin://plugin.video.youtube/play/?video_id=%s" % video_id
+            temp = temp + 1
+            if temp == 1:
+                PlayURL(url)
+                break
+    eod() 
+
+def Real_News_WDK_Sub_Menu(title=''): #The Main Menu
+    WhereAmI('@ Real News Live')
+    
+    url = 'https://www.infowars.com/watch-alex-jones-show/'
+    response = urllib2.urlopen(url)
+    if response and response.getcode() == 200:
+        content = response.read()
+        videos= find_multiple_matches(content,"<iframe(.*?)</iframe>")
+        sources = []
+        temp = 0
+        for entry in videos: 

```

Comment:
Not a show stopper, but maybe a better way to do this could be:
  for temp, entry in enumerate(videos):

That will keep a count in temp and eliminate the need for temp = 0 and temp = temp +1 in your code